lib/liquid/nimble_parser.ex

```diff
@@ -90,6 +92,9 @@ defmodule Liquid.NimbleParser do
   defparsec(:decrement, Decrement.tag())
   defparsec(:increment, Increment.tag())
 
+  defparsecp(:comment_content, Comment.comment_content())
```

Comment:
Isn't `comment_content` already composed by the `comment` parsec?

Reply:
`comment_content` is called recursively